docs/blogs/EnhancedDevPreviewGatewayAPI/GettingStarted.md

```diff
@@ -0,0 +1,178 @@
+# Getting Started
+
```

Comment:
If you haven't read the original blog with an overview of the Gateway API components, please take a minute to review it at `link to IntroGatewayAPI.md`